.github/CODEOWNERS

```diff
@@ -78,7 +78,7 @@
 #/avm/res/network/application-gateway/ @Azure/avm-res-network-applicationgateway-module-owners-bicep @Azure/avm-core-team-technical-bicep
 #/avm/res/network/application-gateway-web-application-firewall-policy/ @Azure/avm-res-network-applicationgatewaywebapplicationfirewallpolicy-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/network/application-security-group/ @Azure/avm-res-network-applicationsecuritygroup-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/network/azure-firewall/ @Azure/avm-res-network-azurefirewall-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/network/azure-firewall/ @Azure/avm-res-network-azurefirewall-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Blocking until Team Parent is configured

Reply:
@matebarabas can you do that?

Reply:
I've created both Teams:

https://github.com/orgs/Azure/teams/avm-res-network-azurefirewall-module-owners-bicep/
https://github.com/orgs/Azure/teams/avm-res-network-azurefirewall-module-contributors-bicep/

Reply:
Parent teams?

Reply:
It is 'avm-technical-reviewers-bicep' for the owners team and 'avm-module-contributors-bicep' for the contributors team, see in the settings of the respective team: 
![image](https://github.com/Azure/bicep-registry-modules/assets/61878316/e2491a6d-63a2-4c1e-aedb-00dd2fe56353)

Reply:
Ok, updated.